source/camera/basic/QuickCaptureTutorial.md

```diff
@@ -26,14 +26,14 @@ camera = app.connect_camera()
 
 Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
 ```python
-frame = camera.capture()
+frame = camera.capture(settings)
```

Comment:
Could it be confusing that we don't mention where "settings" come from?

Reply:
Good catch. I forgot to implement that. Done - check fixup commit.

Reply:
Nice